core/network/miniupnp.cpp

```diff
@@ -43,7 +43,11 @@ bool MiniUPnP::Init()
 		WARN_LOG(NETWORK, "UPnP discover failed: error %d", error);
 		return false;
 	}
+#if (MINIUPNPC_API_VERSION >= 18)
+	error = UPNP_GetValidIGD(devlist, &urls, &data, lanAddress, sizeof(lanAddress), nullptr, 0);
+#else
 	error = UPNP_GetValidIGD(devlist, &urls, &data, lanAddress, sizeof(lanAddress));
+#endif
```

Comment:
Already done in `dev` branch with commit https://github.com/flyinghead/flycast/commit/71982eda7a038e24942921e558845103b6c12326. Do you need these changes in `master` now?

Reply:
Well yes, it would be nice :p